src/runner/test_case_system.py

```diff
@@ -24,7 +32,10 @@ def run(self):
         self.mcl.setup()
 
         # load test case
-        test = self.load_test_case("NoSIM")
+        test_case_name = "NoSIM"
+        if args.t != None:
```

Comment:
Yeah let's fail hard here and tell the user to specify a test case name. Let's do the same thing for config files too!